openshift/templates/secrets/README.md

```diff
@@ -50,22 +50,40 @@ oc process -f database-secrets.yaml -p ENV_NAME=prod | oc create -f -
 cd openshift/templates/app/secrets
 
 oc project $namespace_dev
-oc process -f api-secrets.yaml \
+oc process -f api-sso-secrets.yaml \
   -p ENV_NAME=dev \
   -p KEYCLOAK_AUTHORITY=[sso-dev realm URL] \
   | oc create -f -
 
+oc project $namespace_dev
+oc process -f api-health-secrets.yaml \
+  -p ENV_NAME=dev \
+  -p KEYCLOAK_AUTHORITY=[health-dev realm URL] \
+  | oc create -f -
+
 oc project $namespace_test
-oc process -f api-secrets.yaml \
+oc process -f api-sso-secrets.yaml \
   -p ENV_NAME=test \
   -p KEYCLOAK_AUTHORITY=[sso-test realm URL] \
   | oc create -f -
 
+oc project $namespace_test
+oc process -f api-health-secrets.yaml \
+  -p ENV_NAME=test \
+  -p KEYCLOAK_AUTHORITY=[health-test realm URL] \
+  | oc create -f -
+
```

Comment:
I believe the new health-secrets don't have a "KEYCLOAK_AUTHORITY" parameter... did you mean "ROCKETCHAT_HEALTH_HOOK_URI" by any chance?